src/main/java/uk/ac/ebi/ena/webin/cli/context/transcriptome/TranscriptomeManifestReader.java

```diff
@@ -161,6 +168,9 @@ public TranscriptomeManifestReader(
 			manifest.setTpa( getAndValidateBoolean( getManifestReaderResult().getField(Field.TPA ) ) );
 		}
 
+		manifest.setSubmissionTool(getManifestReaderResult().getValue(ReadsManifestReader.Field.SUBMISSION_TOOL));
+		manifest.setSubmissionToolVersion(getManifestReaderResult().getValue(ReadsManifestReader.Field.SUBMISSION_TOOL_VERSION));
```

Comment:
Should not point to ReadsManifestReader Field/Description. Should use the fields from this class. Ot we could add them into a common class.